CGATPipelines/PipelineMapping.py

```diff
@@ -3044,12 +3044,21 @@ def mapper(self, infiles, outfile):
                 nfiles -= 2
             else:
                 raise ValueError("unexpected number of files")
-            index_prefix = "%(bowtie_index_dir)s/%(genome)s_cs"
+            if exexutable == "bowtie":
+                index_prefix = "%(bowtie_index_dir)s/%(genome)s_cs"
+            else:
```

Comment:
I would even do `elif executable = "bowtie2"` and raise an exception in the `else`, as a sanity check

Reply:
Yes I agree with you. I will make the changes.